### DISCUSSION TITLE
No GP logs parsing

### DISCUSSION THREAD

**Discussion by pzafar1974** [PaloAltoNetworks/Splunk-Apps#175] [General]
Hi,

We are running PanOS 9.1.6 (1.5). I see no data related to Global Protect. Events are not parsing for GP at all. I gave upgraded the app and TA to 6.5.1 but still i see no data parsing. 

I would appreciate your help.

Thanks

**Comment by pzafar1974** [comment]
Please ignore this. Redeploying the latest update fix the issue. Now I can see the logs properly parsing for GP.

**Reply by btorresgil** [reply]
Thanks for the follow up! Glad to hear it's working!
